staging/src/k8s.io/apiserver/pkg/audit/policy/checker.go

```diff
@@ -143,14 +143,29 @@ func ruleMatchesResource(r *audit.PolicyRule, attrs authorizer.Attributes) bool
 
 	apiGroup := attrs.GetAPIGroup()
 	resource := attrs.GetResource()
+	fullResource := ""
```

Comment:
It's not `fullResource`, it's `subresource`, since it's empty when there's no subresource